Installation/cmake/modules/CGAL_add_test.cmake

```diff
@@ -43,6 +43,11 @@ if(ANDROID)
   set(ANDROID_DIR_PREFIX /data/local/tmp/)
   find_program(adb_executable adb)
 endif()
+if(SSH)
```

Comment:
I think that option `SSH` should be renamed to something else.

Reply:
Like EXECUTE_REMOTELY ?

Reply:
I think the name should start with `CGAL_`... maybe:

    CGAL_RUN_TESTS_THROUGH_SSH

Reply:
Ok, I like it, it makes it clear what it does.